### ISSUE TITLE
Word cloud accessibility - use aria-hidden to hide it

### ISSUE THREAD

**Issue by kanarinka** [c4fcm/DataBasic#93]
* use aria-hidden here
Hide the following from the screen reader, as it makes little sense (I believe this may be the word cloud, which is, by its nature, not accessible):
...
What do I do next?
loveknowi'mdon'tyeahohit'swantbabywelli'llcomegotyou'reseegotimegetsaygirllikethat'sonegonnalittlecan'tbackrightlettellsaidneverdayshe'slongheygoodmakewaytake
...

Suggest also hiding the "h3" tag just prior to the word cloud since it seems to make no sense here anyhow.
